### ISSUE TITLE
A11Y tests: missing story names when debugging

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#1071]
## Context

With #952 we have upgraded Storybook.

The new version contained an important breaking change related to story lazy loading: story functions no longer expose a `storyName` property.

Our automated test helpers (`getStories` & `testAccessibility`) relied on `storyName` so when you debug a test, you end up with `Story: undefined`.

We need to adapt to this change to ease debugging.

Note that we have bigger refactoring plans for these helpers and our tests in general (see #963) so this is just a quick fix in the meantime.

**Comment by florian-sanders-cc** [comment]
Fixed with #1072
